common/src/main/java/org/opensearch/sql/common/utils/QueryContext.java

```diff
@@ -6,20 +6,26 @@
 
 package org.opensearch.sql.common.utils;
 
+import java.time.LocalDateTime;
 import java.util.Map;
 import java.util.UUID;
 import org.apache.logging.log4j.ThreadContext;
 
 /**
- * Utility class for generating/accessing the request id from logging context.
+ * Utility class for recording and accessing context for the query being executed.
```

Comment:
Can you add implementation details here, to say that the variables are being stored in the thread context. 
```
 * Implementation Details: context variables is being persisted statically in the thread context (see: @ThreadContext)
```